include/mbedtls/pkcs7.h

```diff
@@ -220,7 +247,7 @@ int mbedtls_pkcs7_signed_data_verify( mbedtls_pkcs7 *pkcs7,
  * \note           This function is different from mbedtls_pkcs7_signed_data_verify()
  *                 in a way that it directly recieves the hash of the data.
  *
- * \return         A negative error code on failure.
+ * \return         0 if the signatures match, or a negative error code on failure.
```

Comment:
There's really only one signature
```suggestion
 * \return         0 if the signature verifies, or a negative error code on failure.
```